src/openrct2/world/Sprite.cpp

```diff
@@ -897,7 +897,7 @@ uint16_t remove_floating_sprites()
         }
         else if (rctSprite->generic.Is<MoneyEffect>())
         {
-            sprite_remove(rctSprite->AsMoneyEffect());
+            sprite_remove(&rctSprite->generic);
```

Comment:
This is on the "Remove AsBalloon" commit, but deals with `MoneyEffect`

Reply:
yeah I didn't see the point converting it only to unconvert it so changed it all at the same time.